src/diffusers/models/cross_attention.py

```diff
@@ -224,7 +254,7 @@ def get_attention_scores(self, query, key, attention_mask=None):
 
         return attention_probs
 
-    def prepare_attention_mask(self, attention_mask, target_length):
+    def prepare_attention_mask(self, attention_mask, target_length, batch_size):
```

Comment:
this argument (`batch_size`) cause the fast test to break - should we make it optional?

Reply:
@patrickvonplaten

Reply:
Good point! I've now added `batch_size` everywhere and deprecated not passing it. Let me know what you think :-)